automation/src/test/resources/reference.conf

```diff
@@ -1,8 +1,8 @@
 leonardo.rImageUrl = "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-r:1.0.13"
-leonardo.pythonImageUrl = "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:0.0.23"
-leonardo.hailImageUrl = "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-hail:0.0.31"
-leonardo.gatkImageUrl = "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:1.0.15"
-leonardo.aouImageUrl = "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:1.0.23"
+leonardo.pythonImageUrl = "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:0.1.1"
```

Comment:
Assuming this will trigger a [custom image](https://fc-jenkins.dsp-techops.broadinstitute.org/job/leonardo-custom-image/) build when merged

Reply:
yep, that's what I'm expecting as well